pix2pix/scripts/get_data.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+import tarfile
+import requests
+
+try:
+    from scripts.helper import get_directory
+except ModuleNotFoundError:
+    from helper import get_directory
+
+url = "https://people.eecs.berkeley.edu/~tinghuiz/projects/pix2pix/datasets/facades.tar.gz"
```

Comment:
@cord add to readme this attribution for facades dataset that we used: https://github.com/phillipi/pix2pix/blob/master/datasets/bibtex/facades.tex

Reply:
Citation is added

Reply:
add  one more for the dataset: http://cmp.felk.cvut.cz/~tylecr1/facade/